src/azul/logging.py

```diff
@@ -20,16 +50,27 @@ def configure_test_logging(*loggers):
 
 
 def _configure_non_app_logging(*loggers):
-    logging.basicConfig(format="%(asctime)s %(levelname)-7s %(threadName)s: %(message)s")
+    non_app_handler = logging.StreamHandler()
+    non_app_handler.addFilter(LambdaLoggerFilter())
+    logging.basicConfig(
+        format='[%(levelname)s]\t%(asctime)s.%(msecs)03dZ\t%(name)s\t%(aws_request_id)s\t%(message)s\n',
```

Comment:
This format string should be extracted into a variable and reused, rather than keeping 2 instances of the same string.